docs/specification/1.0.md

```diff
@@ -7,6 +7,9 @@
 
 The CDS Hooks specification describes the RESTful APIs and interactions between EHRs and CDS Services. All data exchanged through the RESTful APIs MUST BE sent and received as JSON structures, and MUST be transmitted over channels secured using Transport Layer Security (TLS). 
 
+### Terminology
```

Comment:
Suggest the term "Conformance Language" for this section, the word "Terminology" has a very different meaning in this context and is likely to be confused.